man/crypttab.xml

```diff
@@ -327,6 +327,14 @@
       </varlistentry>
 
       <varlistentry>
+        <term><option>tcrypt-veracrypt</option></term>
+
+        <listitem><para>Check for a VeraCrypt volume.  Enabling this
```

Comment:
Also, a short explanation of what VeraCrypt is would be useful. For one, I never heard about it before.